maat-court-data-api/src/main/java/gov/uk/courtdata/entity/CorrespondenceEntity.java

```diff
@@ -0,0 +1,50 @@
+package gov.uk.courtdata.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name = "CORRESPONDENCE", schema = "TOGDATA")
+public class CorrespondenceEntity {
+
+    @Id
+    @Column(name = "ID", nullable = false)
+    private Integer id;
+    @Column(name = "REP_ID", nullable = false)
+    private Integer repId;
+    @Column(name = "GENERATE_DATE", nullable = false)
```

Comment:
As per DB GENERATE_DATE and PRINT_DATE should be LocalDate

Reply:
Keep consistency was used LocalDateTime